exercises/practice/nucleotide-count/nucleotide-count.sql

```diff
@@ -0,0 +1,14 @@
+-- Task:  - Update the nucleotide_count table and set the result based on the input field.
+--        - Update table creation with constraints.
+
+DROP TABLE IF EXISTS nucleotide_count;
+CREATE TABLE "nucleotide_count" (
+    "strand" TEXT,
+    "result" TEXT,
+);
+
+-- Please don't change those import lines:
```

Comment:
Should we also ask they not change the drop/create? Is there a reason that's not done in create_fixture.sql?

Reply:
With SQLite you cannot alter the table and add checks after it was created, so the checks have to come into place when the table is created.

Or you have to rename it, make a new table with the checks, and copy the data.